### DISCUSSION TITLE
Ente

### DISCUSSION THREAD

**Discussion by macb2001** [runtipi/runtipi-appstore#3567] [Requests]
### Description

Open-source and encrypted photo and video backup solution. Alternative to Google Photos.

### Official Website

https://ente.io/

### Documentation link

https://help.ente.io/photos/

### Last application release & date

0.8.94 (May 10, 2024)

### Application license

AGPL-3.0 License

### Source code repository link

https://github.com/ente-io/ente

### Docker image link

ghcr.io/ente-io/server

### Please confirm the following

- [X] I confirm that I've put the app name in the title of the discussion
- [X] I confirm that the application is actively maintained.
- [X] I confirm that the docker image of the application is actively maintained.
- [X] I confirm that I have indeed checked that no similar request for this application already exists

### Gathering crowds

Hey folks!

Please upvote ⬆️ this discussion to show your interest in this request! 

Thanks ⛺

**Comment by Frankynov** [comment]
I would also love to see it running :) project looks very nice but for me the manual installation is a bit too complex for a docker newbie as I am. The Tipi magic would be great here ! :)
